app/stac_api/validators.py

```diff
@@ -526,3 +526,37 @@ def validate_md5_parts(md5_parts, number_parts):
                   'md5 field must be a non empty b64 encoded string'),
                 params={'i': i - 1}, code='invalid'
             )
+
+
+def validate_content_encoding(value):
+    '''Validate the content_encoding field
+
+    Args:
+        value: str
+            value to validate
+
+     Raises:
+        ValidationError in case of invalid content_encoding value
+    '''
+    # See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding for
+    # supported encoding
+    supported_encoding = set(['gzip', 'br', 'deflate', 'compress'])
```

Comment:
since `deflate` and `compress` are hardly used and e.g. not supported by all browsers: should we restrict it to `gzip` and `br`?

Reply:
As discussed I'll be more restrictive and only allow gzip or br

Reply:
Done